### DISCUSSION TITLE
Nuxt 3 wrapper for useFetch nonsense

### DISCUSSION THREAD

**Discussion by upgreidas** [nuxt/nuxt#16913] [General]
So I created wrapper for useFetch as composable to use for API requests in my application, but it returns same response every time. For example if first request is to /book/1 and second request to /book/2 it will still return response from /book/1. Using useFetch directly works fine. I need custom, reusable function. How can I make this work?

Here's sandbox you can check it out: https://codesandbox.io/s/api-composable-des7gn?file=/app.vue

**Comment by danielroe** [comment]
You need to pass a custom key. See https://github.com/nuxt/framework/pull/4955.
